mobile/test/java/io/envoyproxy/envoymobile/engine/testing/HttpProxyTestServerFactory.java

```diff
@@ -0,0 +1,38 @@
+package io.envoyproxy.envoymobile.engine.testing;
```

Comment:
wasn't this part of the previous PR?  I think tip-of-tree needs to be sync'ed with this branch?

Reply:
This is for the *proxy* test server.